Corona-Warn-App/src/main/java/de/rki/coronawarnapp/bugreporting/censors/submission/PcrQrCodeCensor.kt

```diff
@@ -1,13 +1,14 @@
-package de.rki.coronawarnapp.bugreporting.censors
+package de.rki.coronawarnapp.bugreporting.censors.submission
 
 import dagger.Reusable
+import de.rki.coronawarnapp.bugreporting.censors.BugCensor
 import de.rki.coronawarnapp.bugreporting.censors.BugCensor.Companion.toNewLogLineIfDifferent
 import de.rki.coronawarnapp.bugreporting.debuglog.LogLine
 import de.rki.coronawarnapp.util.CWADebug
 import javax.inject.Inject
 
 @Reusable
-class QRCodeCensor @Inject constructor() : BugCensor {
+class PcrQrCodeCensor @Inject constructor() : BugCensor {
```

Comment:
Hm, we also need to censor `CoronaTest.identifier`, as it contains the QRCode code, i.e.

```
PCRCoronaTest(identifier=qrcode-PCR-D61AC9-D61AC989-F49E-4152-880F-1AABB6716CFD, ...)
```
same thing for `RACoronaTest`.

Reply:
fixed.